### PR TITLE
fix, author dropdown positioning, refs steemit/condenser #1533

### PR DESCRIPTION
**Pull request by gl2748**
# Before:
before

# After:
after

### CODE REVIEW

**Comment by gl2748** [comment]
Things that still need to be done.

- [ ] Strip react-foundation-components dependency from here: https://github.com/steemit/condenser/blob/master/app/components/pages/UserProfile.jsx#L22

- [ ] Need to make the overlay close on any-click outside the overlay div element. Like this:
https://github.com/steemit/condenser/blob/master/app/components/elements/FoundationDropdown.jsx#L37-L44

**Comment by sneak** [comment]
Lookin good.  We have this same issue with the user action menu dropdown on mobile.

**Comment by sneak** [comment]
FYI we use four-space indents

**Comment by valzav** [comment]
@gl2748, great work! Any chance you could refactor this into a separate component so we could replace LinkWithDropdown with it in other places?

**Comment by gl2748** [comment]
@sneak @valzav,
Yes, I'll work on this over the weekend and revise the PR accordingly.
Thanks for all your feedback.
Iain.

**Comment by roadscape** [comment]
@gl2748 are you still willing to make these changes or should we merge as-is?

**Comment by gl2748** [comment]
@roadscape yes! I'm going to chat with Ned today at somepoint. I'll keep you posted.

**Comment by gl2748** [comment]
@valzav @roadscape,

I have been working through this.  Please have a look at my approach. I am not sure of the need for a new component, the `Overlay `component offered by `react-overlays` is fairly powerful by itself. Info here:
https://react-bootstrap.github.io/react-overlays/#overlay

What I have done is taken the author dropdown and turned it into a stateless-functional-component (SFC). Perhaps this is the best way forward:

Basically:
Using the Overlay component where necessary and passing it the appropriate contents as a `SFC`.

If you agree, I can continue this line of work refactoring components to remove dependencies on the `react-foundation-components`  next up would be the `FoundationDropdown` component ...

**Comment by gl2748** [comment]
Today I have moved onto removing `react-foundation-components` dependency from the `Author` page.

**Comment by roadscape** [comment]
@gl2748 looking good. Could you please look into why the styles changed?

Before:
image

After:
image

**Comment by roadscape** [comment]
Also could you please rebase on master?

**Comment by gl2748** [comment]
@roadscape I've resolved those issues with the style creeping, rebased on master and split out further work on removing react-foundation-components  into another branch that'll I'll have a PR for shortly. Screen caps below for the updates to match Prod.
### Before.
steembefore

### After.
steemafter

**Comment by roadscape** [comment]
Wonderful, thanks. Be aware we have an unusually large set of changes to merge into master soon (`rc` branch). I might merge this PR first and fix conflicts in the other branch.

File: app/components/elements/Icon.scss [hunk 1]
```diff
@@ -4,7 +4,6 @@
   > svg, span.icon {
     width: 1.12rem;
     height: 1.12rem;
-    vertical-align: top;
```
Comment (roadscape) [inline app/components/elements/Icon.scss]: Are we sure it's safe to get rid of this?

Comment (gl2748) [inline app/components/elements/Icon.scss]: @roadscape,  Yes, I am confident it is safe to remove that.
